tensorboard/defs/defs.bzl

```diff
@@ -68,3 +68,14 @@ def tf_ng_web_test_suite(runtime_deps = [], bootstrap = [], deps = [], **kwargs)
         ],
         **kwargs
     )
+
+def tf_svg_bundle(name, srcs, output_name):
```

Comment:
Optional: Maybe rename `output_name` to `out` for consistency with
native rules and `tensorboard_typescript_bundle`? It’s also shorter. :-)